app/views/characters/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Character#index</h1>
+<h3> All Characters <h3>
+<h2><%= link_to "New Character", new_character_path, method: :new%> </h2>
```

Comment:
`method:` takes an HTTP method like `:patch` or `:delete`. If you're linking to a `new` form, there's no need to specify a `method:`.

Can just do `<%= link_to "New Character", new_character_path %>`